realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncUserTests.java

```diff
@@ -79,6 +99,49 @@ public void setUp() {
         SyncManager.reset();
     }
 
+    private static SyncUser createFakeUser(String id) {
+        final Token token = new Token("token_value", id, "path_value", Long.MAX_VALUE, null);
+        try {
+            return SYNC_USER_CONSTRUCTOR.newInstance(token, authUrl);
+        } catch (InstantiationException e) {
```

Comment:
We are using Java 8 here.  You can say `catch InstantiationException | IllegalAccessException | ... `